packages/terra-application-utility/package.json

```diff
@@ -24,15 +24,13 @@
   "peerDependencies": {
     "react": "^16.4.2",
     "react-dom": "^16.4.2",
-    "terra-app-delegate": "^2.0.0",
     "terra-base": "^3.0.0"
   },
   "dependencies": {
     "classnames": "^2.2.5",
     "prop-types": "^15.5.8",
     "react-intl": "^2.4.0",
-    "terra-app-delegate": "^2.7.0",
-    "terra-application-header-layout": "^2.19.0",
+    "terra-application-header-layout": "^2.17.0",
```

Comment:
Let's set keep this as `^2.19.0`

Reply:
I think the merge with master got jacked after the release. I'll go through the rest of the package updates and make sure nothing got bumped down for no reason.